### PR TITLE
[Docker-Gen] Implement RUN, CMD and EXPOSE commands

### PR DESCRIPTION
**Pull request by Lewiky**
Add support for generation of `RUN`, `CMD` and `EXPOSE` commands in Dockerfile. 

Agreed that we're only going to support one variant of commands with multiple options, as they're all mostly equal anyway.

Unit tests included

### CODE REVIEW

File: src/main/scala/temple/generate/docker/ast/Statement.scala [hunk 1]
```diff
@@ -3,7 +3,14 @@ package temple.generate.docker.ast
...
-  case class From(image: String, tag: Option[String])         extends Statement
-  case class RunCmd(command: String)                          extends Statement
-  case class RunExec(executable: String, params: Seq[String]) extends Statement
+  case class From(image: String, tag: Option[String])     extends Statement
+  case class Run(executable: String, params: Seq[String]) extends Statement
+  case class Cmd(executable: String, params: Seq[String]) extends Statement
+
+  case class Expose(port: Int) extends Statement {
+    {
+      val MAX_PORT = 65500
```
Comment (c-harding) [inline src/main/scala/temple/generate/docker/ast/Statement.scala]: 65535

**Review by c-harding — APPROVED** [review]
lgtm

File: src/main/scala/temple/generate/docker/ast/Statement.scala [hunk 2]
```diff
@@ -3,7 +3,14 @@ package temple.generate.docker.ast
...
-  case class RunCmd(command: String)                          extends Statement
-  case class RunExec(executable: String, params: Seq[String]) extends Statement
+  case class From(image: String, tag: Option[String])     extends Statement
+  case class Run(executable: String, params: Seq[String]) extends Statement
+  case class Cmd(executable: String, params: Seq[String]) extends Statement
+
+  case class Expose(port: Int) extends Statement {
+    {
+      val MAX_PORT = 65535
+      if (!(0 to MAX_PORT contains port)) throw new IllegalArgumentException("EXPOSE requires a valid port")
```
Comment (jaylees14) [inline src/main/scala/temple/generate/docker/ast/Statement.scala:13]: Should this be `1`, since `0` is a reserved port?

**Review by jaylees14 — APPROVED** [review]
👍 LGTM

**Pull request merged**